### ISSUE TITLE
Add verify subcommand for SPARQL queries

### ISSUE THREAD

**Issue by cmungall** [ontodev/robot#177]
See #159 for full details.

Use case: https://github.com/geneontology/go-ontology/pull/14033

we have an ugly perl wrapper over query to do this now, and it's not very informative:

```

../util/check-violations-sparql-output.pl reports/equivalent-classes-violation.csv reports/trailing-whitespace-violation.csv reports/owldef-self-reference-violation.csv
 // in reports/trailing-whitespace-violation.csv
Violations: 1
```

**Comment by dougli1sqrd** [comment]
Currently the sparql queries run and the results are output in CSV in the reports dir. Then the perl script reads them. It doesn't seem like there is a limit on the number of results in terms of violations that can occur, is that true? So if there's millions of violations we end up with a huge report? Should we limit that in the sparql query side, or in the Robot end?

**Comment by cmungall** [comment]
Let's default to 100, and have an option to add more

**Comment by jamesaoverton** [comment]
Closed by #179
